packages/app/src/pages/SwapPage/index.tsx

```diff
@@ -1,36 +1,74 @@
-import type { CoinQuantity } from "fuels";
 import { useState } from "react";
 import toast from "react-hot-toast";
 import { MdSwapCalls } from "react-icons/md";
 import { useMutation, useQuery } from "react-query";
-import { useNavigate } from "react-router-dom";
 
 import { SwapComponent } from "./SwapComponent";
 import { queryPreviewAmount, swapTokens } from "./queries";
 import type { SwapState } from "./types";
-import { ActiveInput } from "./types";
 
 import { Button } from "~/components/Button";
 import { Card } from "~/components/Card";
 import { useContract } from "~/context/AppContext";
-import { useBalances } from "~/hooks/useBalances";
 import useDebounce from "~/hooks/useDebounce";
-import { sleep } from "~/lib/utils";
-import { Pages } from "~/types/pages";
+import { isSwayInfinity, sleep } from "~/lib/utils";
 
-const getBalanceAsset = (
-  balances: CoinQuantity[] | undefined,
-  assetId: string
-) => balances?.find((item) => item.assetId === assetId);
+type StateParams = {
+  swapState: SwapState | null;
+  previewAmount: bigint | null;
+  hasLiquidity: boolean;
+};
+
+enum ValidationStateEnum {
+  SelectToken = 0,
+  EnterAmount = 1,
+  InsufficientBalance = 2,
+  InsufficientLiquidity = 3,
+  Swap = 4,
+}
+
+const getValidationText = (
+  state: ValidationStateEnum,
+  swapState: SwapState | null
+) => {
+  switch (state) {
+    case ValidationStateEnum.SelectToken:
+      return "Select token";
+    case ValidationStateEnum.EnterAmount:
+      return "Enter amount";
+    case ValidationStateEnum.InsufficientBalance:
+      return `Insufficient ${swapState?.coin.symbol || ""} balance`;
+    case ValidationStateEnum.InsufficientLiquidity:
+      return "Insufficient liquidity";
+    default:
+      return "Swap";
+  }
+};
+
+const getValidationState = ({
+  swapState,
+  previewAmount,
+  hasLiquidity,
+}: StateParams): ValidationStateEnum => {
+  if (!swapState?.to || !swapState?.from)
+    return ValidationStateEnum.SelectToken;
+  if (!swapState?.amount) {
+    return ValidationStateEnum.EnterAmount;
+  }
+  if (!swapState.hasBalance) {
+    return ValidationStateEnum.InsufficientBalance;
+  }
+  if (!hasLiquidity || isSwayInfinity(previewAmount))
+    return ValidationStateEnum.InsufficientLiquidity;
```

Comment:
some `if`s have `brackets` and others don't. Either add brackets to all, or remove from all and keep only 1 line

Reply:
it happens because of the auto format we have